computedfont/plugin.js

```diff
@@ -103,7 +103,15 @@
 				for ( var i = 0; i < names.length; i++ ) {
 					name = names[ i ];
 					// Add the tag entry to the panel list.
-					this.add( name, styles[ name ].buildPreview(), name );
+					var preview;
+
+					if (previews !== undefined && previews[ name ] !== undefined) {
+						preview = "<img src='" + previews[ name ] + "' style='padding: 0; margin: 0; margin-bottom: -4px; outline: none;' />";
```

Comment:
Would be nice if it can be made configurable with CSS in future. Maybe adding a style class would suffice for now?